__main__.py

```diff
@@ -3,81 +3,154 @@
 import datetime as dt
 import json
 import sys
+from enum import Enum, auto
 from pathlib import Path
-from typing import NoReturn
+from typing import TYPE_CHECKING, Any, ClassVar, NewType, NoReturn, TypedDict, cast
 
 import requests
 from dahlia import dahlia, dprint
 
+if TYPE_CHECKING:
+    from typing_extensions import Self
+
+
+__version__ = "0.1.0"
+
+
+class Status(str, Enum):
+    def _generate_next_value_(  # type: ignore
```

Comment:
What type-checker error is being suppressed?

Reply:
I don't remember exactly, but I think it's related to the one in the comment:
- https://github.com/python/mypy/issues/7591